src/config_models/cli_args.rs

```diff
@@ -254,6 +254,15 @@ pub struct Args {
     /// Exposing the data directory leaks some privacy. Disable to prevent.
     #[clap(long)]
     pub disable_cookie_hint: bool,
+
+    /// The duration (in seconds) during which new connection attempts from peers
+    /// are ignored after a connection to them was closed.
+    ///
+    /// Does not affect abnormally closed connections. For example, if a connection
+    /// is dropped due to networking issues, an immediate reconnection attempt is
+    /// not affected by this cooldown.
+    #[clap(long, default_value = "30", value_parser = duration_from_seconds_str)]
```

Comment:
Peer discovery runs every 2 minutes which is where a reconnect would happen. So with such a low default value, I don't think current behavior will change noticeably. I would lean towards a much longer default value. How about 30 minutes?